ludwig/models/model.py

```diff
@@ -936,8 +967,8 @@ def update_output_stats_batch(
                             result[field_name][stat_config['output']].sum()
                         )
                         seq_set_size[field_name][stat] = (
-                            seq_set_size[field_name].get(stat, 0) +
-                            len(result[field_name][stat_config['output']])
+                                seq_set_size[field_name].get(stat, 0) +
+                                len(result[field_name][stat_config['output']])
```

Comment:
Move closing parenthesis to the line below.

Reply:
that right parenthesis closes len(), so ti should be alright I believe